components/inputs/input-text.js

```diff
@@ -280,6 +286,11 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
+		const input = this.shadowRoot.querySelector('.d2l-input');
+		if (input) {
```

Comment:
I think this element will always exist at this point.  Just wondering if I am missing something...?

Reply:
Correct. I've changed this around so that this just calls into `_setValue` which will do this same thing and also set up the validation state the first time.